packages/cmf/src/sagas/http.js

```diff
@@ -117,26 +117,29 @@ export function httpFetch(url, config, method, payload) {
 	if (payload instanceof FormData) {
 		body = payload;
 		delete defaultHeaders['Content-Type'];
-	} else {
+	}
+
+	const params = merge(
+		{
+			credentials: 'same-origin',
+			headers: defaultHeaders,
+			method,
+		},
+		{
+			...HTTP.defaultConfig,
+			...config,
+		},
+	);
+
+	const type = params.headers['Content-Type'];
+	if (type && type.includes('json')) {
```

Comment:
AFAIK, the only content type is `application/json`

Reply:
I don't understand your point. There are tens of HTTP valid content-type, so even if Talend recommends to use `application/json`, I don't see any reason to forbids the use of the other types.
`application/json` is already the default content-type, I guess this is the maximum we could do for that.